content/blog/open-sourcing-workshops-on-demand-part-5-create-a-workshop.md

```diff
@@ -0,0 +1,226 @@
+4. ###### Copy over the workshop developer's home folder a workshop template containing examples of introduction, conclusion and lab notebooks, allowing him to start his work.
+5. **Give the developer the wod-notebook repo url for him to fork the repo and work locally on his machine (when workshop does not require appliance but just a jupyter kernel for instance)**
+6. **Whenever ready, a pull request can be made. The admin can then review and accept it. The admin can then perform the necessary steps to prepare the infrastructure to host the workshop**
+
+
+
+##### O﻿n the database server:
+
+In order to exist, a workshop requires serveral entries in the database:
+
+I﻿n the **Workshops table:**
+
+A﻿ new entry will need the following:
+
+![](/img/wod-db-go-1.png "Workshop's fields in the Database.")
+
+* **An id:** A workshop id to be used by backend server automation and Replays table to reference the associated replay video of the workshop.
+* **A﻿ name:** The workshop's name as it will will be displayed on the registration portal.
+* **A name of the folder** containing all the workshop's notebooks
+* **A﻿ description / abstract**
+* **A﻿ capacity:** The number of maximum concurrent students allowed to take on the workshop.
+* **A﻿ student range:** The range between which students get picked at registration time.
+* **R﻿eset and ldap** entries are to be used by backend server automation if dedicated reset scripts and ldap authentication are required by the workshop.
+* **A﻿ session type:** Workshops-on-Demand by default
+* **A﻿ location:** If your setup includes multiple production sites, use this field to allocate workshops according to your needs. In the case of the HPE Developer Community, some workshops can only run on a HPE GreenLake cloud environment. As a consequence, the location is set to greenlake in this case.
+* **A﻿vatar, role and replayLink** are superseeded by entries in the replay table. I will explain later.
+
+![](/img/wod-db-go-2.png "Workshop's fields in the Database #2.")
+
+* **Compile:** This entry will be filled with the name of a script to be compiled at deployment time. This feature allows for instance the admin to hide login scripts and credentials in non-editable executable files.
+* **Varpass:**  This defines whethere a workshop require some password variable to be leveraged or not.
+* **R﻿eplayId:** This entry links the dedicated replay video to the workshop. it enables the presence of the replay in the learn more page of the workshop.
+* **W﻿orkshopImg:** As part of the lifecycle of the workshop, several emails are sent to the student. A workshop image is embbeded in the first emails.
+* **B﻿adgeImg:** As part of the lifecycle of the workshop, several emails are sent to the student. In the final email, a badge is included. It allows the student to share its accomplishment on SoME like linkedin for instance.
```

Comment:
I would change "SoMe" here to "social media". Not everyone in engineering is familiar with the SoMe acronym.